apptools/preferences/preferences_helper.py

```diff
@@ -99,8 +99,11 @@ def _anytrait_changed(self, trait_name, old, new):
         # If the change refers to a trait defined on this class, then
         # the trait is not a preference trait and we do nothing.
 
-    def _preferences_changed(self, old, new):
-        """ Static trait change handler. """
+    @observe("preferences", post_init=True)
+    def _preferences_changed(self, event):
```

Comment:
Hmm. To avoid confusion, the name of this method should be changed away from the magic method name.
```suggestion
    def _update_preferences_listeners(self, event):
```

Reply:
Fixed in [6cb93e8](https://github.com/enthought/apptools/pull/343/commits/6cb93e8ba3d086650be9bee13a111205df5d0a34)